recipe/meta.yaml

```diff
@@ -51,7 +50,7 @@ outputs:
         # does not need to link to tbb).
         # pagmo's API and ABI change at minor rev bumps.
         - {{ pin_subpackage('pagmo', max_pin='x.x') }}
-        - {{ pin_compatible('boost-cpp', max_pin='x.x') }}
+        - {{ pin_compatible('libboost-devel', max_pin='x.x') }}
```

Comment:
This is running against a limitation of not being able to preview all eventualities in the renaming migrator. Aside from the fact that it's unusual to run-export third-party libraries, this should now use `libboost` as a run-export, because `-devel` brings along 15k headers that users of a pagmo-dependee do not need at runtime.